.gitignore

```diff
@@ -17,7 +17,7 @@ packages/*/*/assets
 packages/*/*/images
 packages/*/*/rhel-*
 packages/*/*/ubuntu-*
-scripts/Manifest
```

Comment:
@laverya Thinking better that is not the best solution. Why?
If we need to change the layout we will need to push 2 prs ( one removing it and with the change then another returning back, if we forget the second one we might face the error again )

Reply:
you can still commit things that are in gitignore, you just need to use the right commands

Reply:
hmmm apparently gitignore does not work like I remembered it did

Reply:
there's update-index --assume-unchanged but that's just for local